src/main/java/sirius/biz/protocol/Protocols.java

```diff
@@ -140,6 +145,47 @@ public void handle(Incident incident) throws Exception {
         }
     }
 
+    private String truncateStacktrace(Incident incident) {
+        boolean foundStackTrace = false;
+        StringBuilder result = new StringBuilder();
+        StringBuilder stack = new StringBuilder();
+        StringBuilder errorMesssage = new StringBuilder();
+
+        for(String line: NLS.toUserString(incident.getException()).split("\\r?\\n")) {
```

Comment:
Code format! (No space between `for` and the bracket.) – The check boxes are there to remind you of that, rather than to be checked automatically. 😜

Reply:
I would prefer `NLS.toUserString(incident.getException()).lines().forEach(line -> {…});` as it saves the non-compiled regular expression.

Reply:
That would also allow to use early returns instead of the nested `ifs` and `elses`.